Input/Input.module.scss

```diff
@@ -0,0 +1,290 @@
+// Input.module.scss
+//
+@use "../styles/color-mixins.scss";
+@use "../styles/colors.scss";
+@use "../styles/mixins.scss";
+@use "../styles/skin.scss";
+@use "../styles/variables.scss";
+
+.inputPopupWrapper {
+	display: inline;
+}
+
+.popup {
+	.numberField,
+	.number,
+	.password,
+	.separated {
+		/* Available for customization */
+	}
+
+	.back {
+		position: absolute;
+
+		@include mixins.sand-custom-text-large {
+			@include mixins.sand-small-button-text;
+		}
+	}
+
+	.body {
+		line-height: 0; // This removes an undesirable layout shift occurring when a number is entered into the numberCells (Popup adds a line-height to all children).
+	}
+
+	.titles {
+		max-width: 100%;
+		width: 100%;
+	}
+
+	.fieldWrapper {
+		display: inline-block;
+		position: relative;
+	}
+
+	.joined,
+	.numberCell {
+		position: relative;
+		z-index: 0;
+
+		&::before {
+			position: absolute;
+			@include mixins.position(0);
+			content: '';
+			z-index: -1;
+
```

Comment:
Please remove empty line. Also, don't the properties after @include need to be wrapped in '&'?